code/index.html

```diff
@@ -1,30 +1,148 @@
+    <main>
+        <!-- FORM -->
+        <section class="form-parent">        
+            <form action="http://httpbin.org/anything" method="POST">
+                <div class="input-section">
+                    <h3>Subscribe to the house plant hotel newsletter</h3>
+                    <label for="fname">
+                        <input type="text" id="fname" name="fname" placeholder="First name" required>
+                    </label>
+                    <label for="lname">
+                        <input type="text" id="lname" name="lname" placeholder="Last name" required>
+                    </label>
+                    <label for="email">
+                        <input type="email" id="email" name="email" placeholder="Email" required>
+                    </label>
+                </div>
+                
+                <!-- RADIO BUTTON SECTION -->
+                <div class="radio-button-column">
+                    <h3>Interested in events at the plant house hotel?</h3>
+                    <div class="radio-button-row">
+                        <label class="radio-button-container">
+                            <input type="radio" id="yes" name="answer" checked required> 
+                            Yes please
+                            <span class="custom-radio"></span>
+                        </label>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+                        <label class="radio-button-container">
+                            <input type="radio" id="no" name="answer" required>  
+                            No thanks
+                            <span class="custom-radio"></span>
+                        </label>
+                    </div> 
+                </div>
+            
+                <!-- CHECKBOX SECTION -->
+                <div class="checkbox-column">
+                    <div class= "checkbox-row">
+                    <label class="checkbox-container">
+                        <input type="checkbox" id="terms" name="terms" checked required>
+                        I agree to terms & conditions
+                        <span class="custom-checkbox"></span>
+                    </label>
+                    </div>
+                </div>
+
+                <div class="submit-container">
+                    <input type="submit" name="submit" value="SUBMIT">
+                </div>
+            </form>
+        </section>
+        
+        <!-- ACCORDION -->
+        <section class="accordion-container">
+            <h3>Frequently asked questions</h3>
+
+            <div class="question" id="section1">
+                <span>What kind of plant feed do you feed the plants?</span>
+                <span class="pointer">▼</span>
+            </div>
+            <div class="answer">
+                <p>We used top grade 100% organic, vegan plant food.</p>
+            </div>
```

Comment:
You could debate naming these classes `accordion-question` and `accordion-answer` if you wanted your CSS file to be more self-organizing